lib/EvalDevToolModuleTemplatePlugin.js

```diff
@@ -2,28 +2,33 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var RawSource = require("webpack-sources").RawSource;
-var ModuleFilenameHelpers = require("./ModuleFilenameHelpers");
+"use strict";
 
-function EvalDevToolModuleTemplatePlugin(sourceUrlComment, moduleFilenameTemplate) {
-	this.sourceUrlComment = sourceUrlComment || "\n//# sourceURL=[url]";
-	this.moduleFilenameTemplate = moduleFilenameTemplate || "webpack:///[resourcePath]?[loaders]";
+const RawSource = require("webpack-sources").RawSource;
+const ModuleFilenameHelpers = require("./ModuleFilenameHelpers");
+
+class EvalDevToolModuleTemplatePlugin {
+	constructor(sourceUrlComment, moduleFilenameTemplate) {
+		this.sourceUrlComment = sourceUrlComment || "\n//# sourceURL=[url]";
+		this.moduleFilenameTemplate = moduleFilenameTemplate || "webpack:///[resourcePath]?[loaders]";
+	}
+
+	apply(moduleTemplate) {
+		const self = this;
+		moduleTemplate.plugin("module", function(source, module) {
```

Comment:
Rewrite to array function to remove `self`.

`this` can be replaced with `moduleTemplate`